lighthouse-core/lib/file-namer.js

```diff
@@ -16,11 +16,11 @@
  * Generate a filenamePrefix of hostname_YYYY-MM-DD_HH-MM-SS
  * Date/time uses the local timezone, however Node has unreliable ICU
  * support, so we must construct a YYYY-MM-DD date format manually. :/
- * @param {{finalUrl: string, fetchTime: string}} lhr
+ * @param {{requestedUrl: string, finalUrl: string, fetchTime: string}} lhr
```

Comment:
What is this requestedUrl business about?  Does it relate to INSECURE DOC requests?

Reply:
Oof you're right, forgot to check if that was still necessary.